README.md

```diff
@@ -80,7 +80,7 @@ make install
 Execute following command in root directory.
 
 ```
-./example/vgg16_example_in_cpp
+./example/mkldnn_vgg16_example_in_cpp // for tensorrt, tensorrt_vgg16_example_in_cpp is available
```

Comment:
`//` is still used here, despite the changes of https://github.com/pfnet-research/menoh/pull/214/commits/573c985ffff30dc5b8b17ea64fa5bac9ab97168e.

Also, I suggested that "tensorrt backend" might be better than  just "tensorrt" here.
However https://github.com/pfnet-research/menoh/pull/214/commits/573c985ffff30dc5b8b17ea64fa5bac9ab97168e did not change here, but it replaces "tensorrt backend" in different place with "tensorrt". I do not object about it, but I'd like to confirm just in case.

Reply:
OK. I will consolidate tensorrt backend `tensorrt` and fix here as well.